omod/src/main/webapp/resources/jsonforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.json

```diff
@@ -50,7 +50,6 @@
             {
               "label": "Pregnancy",
```

Comment:
Be consistent with the name of the form https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/6db31049fa3b82eb6b474e3121890cae04c93de6/omod/src/main/webapp/resources/jsonforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.json#L2 ensure its same as the old HTML formentry